dist/images/ovndb-raft-functions.sh

```diff
@@ -223,29 +230,44 @@ ovsdb-raft() {
   else
     database="OVN_Southbound"
     [[ "yes" == ${OVN_SSL_ENABLE} ]] && {
+      echo "BILLY: SB SSL Enabled"
       db_ssl_opts="
             --ovn-sb-db-ssl-key=${ovn_sb_pk}
             --ovn-sb-db-ssl-cert=${ovn_sb_cert}
             --ovn-sb-db-ssl-ca-cert=${ovn_ca_cert}
       "
     }
   fi
+
+  # 'ovn-${db}ctl set-connection' called in ovsdb-raft() defaults to IPv4.
+  # For IPv6, overwrite with ":[::}"
+  ovn_raft_conn_ip=""
+  if [[ "${ovn_db_host}" == *":"* ]]; then
+    ovn_raft_conn_ip=":[::]"
+  fi
+  echo "BILLY: Search ovn_db_host=${ovn_db_host} for : -- ovn_raft_conn_ip=${ovn_raft_conn_ip}"
+
+
   if [[ "${POD_NAME}" == "ovnkube-db-0" ]]; then
+    echo "BILLY: ovnkube-db-0 - ovn_db_host=${ovn_db_host} db_ssl_opts=${db_ssl_opts}"
     run_as_ovs_user_if_needed \
       ${OVNCTL_PATH} run_${db}_ovsdb --no-monitor \
-      --db-${db}-cluster-local-addr=${ovn_db_host} \
+      --db-${db}-cluster-local-addr=[${ovn_db_host}] \
```

Comment:
normally square brackets are only to build an `[ip]:port` url, for standalone addresses should not be used, just a comment because don't have any idea how ovs process it :smile: 
I didn't add square brackets for my local test and worked

Reply:
Done

Reply:
Undone. I found I needed this with my testing:
```
$ kubectl describe pod -n ovn-kubernetes ovnkube-db-0
Name:         ovnkube-db-0
Namespace:    ovn-kubernetes
Priority:     0
:
Containers:
  nb-ovsdb:
    Container ID:  containerd://91748e38521930bc34ebc72e59692884fa418cdf2e6522582594c85ff1bcb053
    Image:         docker.io/library/ovn-daemonset-f:dev
    Image ID:      sha256:5053328cd57a96c976312a8d30eff7c3530974d4d1c5640c81ae93ced8889f11
    Port:          <none>
    Host Port:     <none>
    Command:
      /root/ovnkube.sh
      nb-ovsdb-raft
    State:       Running
      Started:   Tue, 09 Jun 2020 17:43:58 -0400
    Last State:  Terminated
      Reason:    Error
      Message:   run/ovn/ovnnb_db.sock: connecting...
2020-06-09T21:41:42Z|00002|reconnect|INFO|unix:/var/run/ovn/ovnnb_db.sock: connection attempt failed (No such file or directory)
ovsdb-tool: ovsdb error: tcp:fc00:f853:ccd:e793::3:6643: syntax error in address
Creating cluster database /etc/ovn/ovnnb_db.db ... failed!
:
  sb-ovsdb:
    Container ID:  containerd://879f09c4a57c86535fc038dcc4bebee65d14fc14163ac22ff3acf88ae5319284
    Image:         docker.io/library/ovn-daemonset-f:dev
    Image ID:      sha256:5053328cd57a96c976312a8d30eff7c3530974d4d1c5640c81ae93ced8889f11
    Port:          <none>
    Host Port:     <none>
    Command:
      /root/ovnkube.sh
      sb-ovsdb-raft
    State:       Running
      Started:   Tue, 09 Jun 2020 17:43:58 -0400
    Last State:  Terminated
      Reason:    Error
      Message:   run/ovn/ovnsb_db.sock: connecting...
2020-06-09T21:41:42Z|00002|reconnect|INFO|unix:/var/run/ovn/ovnsb_db.sock: connection attempt failed (No such file or directory)
ovsdb-tool: ovsdb error: tcp:fc00:f853:ccd:e793::3:6644: syntax error in address
Creating cluster database /etc/ovn/ovnsb_db.db ... failed!
```

Reply:
Wow, nice catch